src/act/node/node_parts/mod.rs

```diff
@@ -1,3 +1,4 @@
 pub mod member;
+pub mod mode;
```

Comment:
I would bet that Dan would ask to add `pub use mode::Mode` and for that matter we should also add `pub use member::Member` etc.